source/Plx.js

```diff
@@ -897,4 +922,6 @@ Plx.defaultProps = {
   parallaxData: [],
   style: {},
   tagName: 'div',
+  onPlxStart: emptyFunction,
+  onPlxEnd: emptyFunction,
```

Comment:
Why not `null`? IMHO `this.props.onPlxStart !== null` is a bit cleaner check.

Reply:
True.
I'd done this to keep one flag "callbacksEnabled" and then fire both if even one prop (ex. onlt onPlxStart and not onPlxEnd) is defined. I'm changing the code now to have two flags, plxStartEnabled and plxEndEnabled, so that we can fire only the applied prop.